protocol-dashboard/src/store/cache/analytics/hooks.ts

```diff
@@ -369,14 +372,14 @@ const getTrailingAPI = async () => {
 }
 
 export function fetchTrailingApiCalls(
-  bucket: Bucket
+  bucket: Bucket.MONTH | Bucket.WEEK | Bucket.YEAR
```

Comment:
it might make sense to use the "TrailingApiCallsBucket" here? if so not sure best place to have that type, but just an idea. additionally probably also want to reuse for `getTrailingAPI` and `useTrailingApiCalls` as well.